src/app/components/new-goal/new-goal.component.html

```diff
@@ -1,54 +1,53 @@
-            <mat-datepicker #picker1></mat-datepicker>
-        </mat-form-field>
-
-        <mat-form-field appearance="outline">
-            <mat-label>Due Date</mat-label>
-            <input matInput [matDatepicker]="picker2" placeholder="Choose a date to end"
-            (dateInput)="onEndDate($event)">
-            <mat-datepicker-toggle matSuffix [for]="picker2"></mat-datepicker-toggle>
-            <mat-datepicker #picker2></mat-datepicker>
-        </mat-form-field>
-
-    </div>
-
-    <div class="goal-summary">
-      <div class="statics">
-          <div><h2>{{ diffDays }}</h2></div>
-          <div><p>Total Days</p></div>
-      </div>
-      <div class="statics">
-          <div><h2>{{ avgWorkload | number:'1.1-1'}}</h2></div>
-          <div><p>Expected Daily Workload</p></div>
+    <div class="center-row">
+        <form class="center-column">
+          <div class="example-container">
+            <mat-form-field appearance="outline">
+                <mat-label>Name Tag</mat-label>
+                <input matInput placeholder="Placeholder" (input)="onGoalName($event)">
+            </mat-form-field>
+
+            <mat-form-field appearance="outline">
+                <mat-label>Toal Workload</mat-label>
+                <input matInput placeholder="Placeholder" (input)="onGoalWorkload($event)">
+                <mat-hint>*Quantify goals as numbers such as 7 (chapters), 100 (questions)...  </mat-hint>
+            </mat-form-field>
+
+            <mat-form-field appearance="outline">
+                <mat-label>Start Date</mat-label>
+                <input matInput [matDatepicker]="picker1" placeholder="Choose a date to start"
+                       (dateInput)="onStartDate($event)">
+                <mat-datepicker-toggle matSuffix [for]="picker1"></mat-datepicker-toggle>
+                <mat-datepicker #picker1></mat-datepicker>
+            </mat-form-field>
+
+            <mat-form-field appearance="outline">
+                <mat-label>Due Date</mat-label>
+                <input matInput [matDatepicker]="picker2" placeholder="Choose a date to end"
+                (dateInput)="onEndDate($event)">
+                <mat-datepicker-toggle matSuffix [for]="picker2"></mat-datepicker-toggle>
+                <mat-datepicker #picker2></mat-datepicker>
+            </mat-form-field>
+        </div>
+      </form>
+
+      <div class="center-row">
+            <div class="statics">
+                <div><h2>{{ diffDays }}</h2></div>
+                <div><p>Total Days</p></div>
+            </div>
+            <div class="statics">
+                <div><h2>{{ avgWorkload | number:'1.1-1'}}</h2></div>
+                <div><p>Expected Daily Workload</p></div>
+            </div>
       </div>
     </div>
 
-    <button mat-raised-button style="background: #ffb300">Add New Goal</button>
-  </form>
+    <button (click) = "onCreateGoal()" mat-raised-button style="background: #ffb300">Add New Goal</button>
```

Comment:
remove spaces around the = sign

Reply:
Also, avoid inline styles and define them in css files.

Reply:
ok